scale/storage/migrations/0007_auto_20170217_0901.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('storage', '0006_auto_20170127_1423'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='scalefile',
+            name='file_type',
+            field=models.CharField(default='SOURCE', max_length=50, choices=[('SOURCE', 'SOURCE'), ('PRODUCT', 'PRODUCT')]),
+            preserve_default=True,
+        ),
+        migrations.AlterIndexTogether(
+            name='scalefile',
+            index_together=set([('file_type', 'last_modified'), ('file_type', 'data_started', 'data_ended')]),
```

Comment:
I removed the multi-column index. Based on some testing, I think the existing single column indexes are sufficient for now. Changing the index order (ASC, DESC) on a multi-column index is not supported on our version of Django. We need to upgrade as it was recently implemented.